### PR TITLE
feat: add wallet address to published measurements

### PR DESCRIPTION
**Pull request by bajtos**
Connect to:
- https://github.com/filecoin-station/spark/issues/13

### CODE REVIEW

File: spark-publish/index.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const publish = async ({
       id,
       spark_version,
       zinnia_version,
+      -- wallet_address,
```
Comment (juliangruber) [inline spark-publish/index.js:18]: why is this commented out?

Comment (bajtos) [reply spark-publish/index.js:18]: Oops, I wanted to see if there is any failing test and then I forgot to uncomment 🙈

Comment (bajtos) [reply spark-publish/index.js]: Fixed. I also added a test to verify that we are correctly converting the address from Filecoin to Ethereum format.

File: spark-publish/test/test.js [hunk 2]
```diff
@@ -79,6 +79,66 @@ describe('integration', () => {
...
+    await client.query('DELETE FROM measurements')
+
+    const measurementRecorded = {
+      sparkVersion: '1.2.3',
+      zinniaVersion: '0.5.6',
+      cid: 'bafytest',
+      providerAddress: '/dns4/localhost/tcp/8080',
+      protocol: 'graphsync',
+      // We are recording wallet address in Filecoin format (f4...)
+      walletAddress: 'f410ftgmzttyqi3ti4nxbvixa4byql3o5d4eo3jtc43i',
```
Comment (juliangruber) [inline spark-publish/test/test.js:91]: could you please also add a test for testnet addresses?

**Review by juliangruber — CHANGES_REQUESTED** [review]

File: spark-publish/test/test.js (hunk 2 above)
Comment (bajtos) [reply spark-publish/test/test.js:101]: Done - see [fc711f1](https://github.com/filecoin-station/spark-api/pull/89/commits/fc711f16a6f37eb914719bf23437fb07c8216899)

**Review by juliangruber — APPROVED** [review]

**Pull request merged**